cypress/support/commands.js

```diff
@@ -38,7 +38,7 @@ Cypress.Commands.add('clearCookieConsentModal', () => {
   if (consentModal.length > 0) {
     cy.get('.truste_popframe').then((iframe) => {
       const body = iframe.contents().find('body');
-      cy.wrap(body).find('a.call').click()
+      cy.wrap(body).find('a.call').click().wait(2000)
```

Comment:
modal cookie agreement was being accepted, it was an intermittent issue when running cypress tests